generate.py

```diff
@@ -27,6 +29,12 @@ def _str_to_bool(s):
                              'boolean, got {}'.format(s))
         return {'true': True, 'false': False}[s.lower()]
 
+    def _ensure_positive_float(f):
```

Comment:
Ohhh wait I see what you're saying. I didn't notice that the `ArgumentTypeError` indicated automatically the argument to which it was objecting. So the error can just say "yo you need a positive float." Got it got it.